app/renderer/components/Inspector/Actions.js

```diff
@@ -51,6 +51,20 @@ export default class Actions extends Component {
       }
     }
 
+    // Special case for 'updateSettings'
+    if (action.methodName === 'updateSettings') {
+      if (_.isString(args[0])) {
+        try {
+          args[0] = JSON.parse(args[0]);
+        } catch (e) {
+          notification.error({
+            message: t('invalidJson', {json: args[1]}),
```

Comment:
do we know that we have at least 2 args?

Reply:
or it should be args[0]?

Reply:
Good catch, thanks.